### PR TITLE
Mod 9544

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

Added text to the filter section on the award spending page. 
Added icon and icon display

**Technical details:**

**JIRA Ticket:**
[DEV-9544](https://federal-spending-transparency.atlassian.net/browse/DEV-9544)

**Mockup:**
https://bahdigital.invisionapp.com/console/share/DJ8W4N3AVG2/981349871/play

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/search/SearchSidebar.jsx [hunk 1]
```diff
@@ -194,11 +196,18 @@ const SearchSidebar = ({
...
-                <h2 className="sidebar-title">Filters</h2>
+                <div className="sidebar-header_title">
+                    <span className="filter-icon">
+                        <FilterIcon />
+                    </span>
+                    <h2 className="sidebar-title">Filters</h2>
+                </div>
+                <div className="sidebar-paragraph__container">
+                    <div className="sidebar-paragraph">Learn which data elements are associated with each filter <TooltipWrapper icon="info" tooltipPosition="right" tooltipComponent={FilterTooltip} /></div>
+
```
Comment (brianpetway) [inline src/js/components/search/SearchSidebar.jsx:207]: unnecessary whitespace

File: src/js/components/award/shared/InfoTooltipContent.jsx [hunk 2]
```diff
@@ -1217,6 +1217,49 @@ export const CFDASectionInfo = (
     </div>
 );
 
+export const FilterTooltip = (
+    <div className="award-summary-tooltip">
+        <div className="tooltip__title" style={{ textAlign: "left" }}>
```
Comment (brianpetway) [inline src/js/components/award/shared/InfoTooltipContent.jsx:1222]: this inline style can be removed without affecting the appearance of the tooltip

File: src/_scss/pages/search/searchSidebar.scss [hunk 3]
```diff
@@ -8,37 +8,79 @@
...
-            & .sidebar-title {
-                @include h6;
-                @include flex(1 1 auto);
-                color: $color-gray;
-                font-size: rem(19);
-                line-height: rem(21);
-                font-weight: $font-semibold;
-                margin: 0;
-            }
-            & .filter-icon {
```
Comment (brianpetway) [inline src/_scss/pages/search/searchSidebar.scss:32]: something about changes to this class has caused an irregularity in the other instance of this class
20230314AccountFilterIcon

File: src/_scss/pages/search/searchSidebar.scss [hunk 4]
```diff
@@ -8,37 +8,79 @@
...
+                        margin-top: rem(0);
+                        ul li {
+                            display: list-item;
+                          }
+                        ul li:before {
+                            content: " • ";
+                        }
+                    }
                 }
+
```
Comment (brianpetway) [inline src/_scss/pages/search/searchSidebar.scss:81]: unneeded whitespace

File: src/_scss/pages/search/searchSidebar.scss [hunk 5]
```diff
@@ -8,37 +8,79 @@
...
             }
+
```
Comment (brianpetway) [inline src/_scss/pages/search/searchSidebar.scss:83]: unneeded whitespace

File: src/_scss/pages/search/searchSidebar.scss [hunk 6]
```diff
@@ -8,37 +8,79 @@
...
+                display: flex;
+                flex-flow: row;
+                padding-right: rem(4);
+                .sidebar-paragraph {
+                    font-weight: $font-semibold;
+                    font-size: $font-size-14;
+                }
+                .tooltip-wrapper {
+                    display: inline-block;
+                    text-align: bottom;
```
Comment (brianpetway) [inline src/_scss/pages/search/searchSidebar.scss:62]: 'bottom' is not an accepted value for text-align; this can be removed without affecting the appearance of the component

**Comment by brianpetway** [comment]
See inline comments.

But also the Filter icon is now missing from the Search page, in addition to the one on the Federal Account page being very large:
20230314FilterIconMissing

File: src/_scss/components/tooltips/_customTooltipContent.scss [hunk 7]
```diff
@@ -76,6 +76,9 @@
         font-style: italic;
     }
 }
+.award-summary__new-catagory{
```
[the remark this answers is not in the document]
Comment (chasls2) [reply src/_scss/components/tooltips/_customTooltipContent.scss:79]: done

File: src/js/components/search/SearchSidebar.jsx (hunk 1 above)
Comment (chasls2) [reply src/js/components/search/SearchSidebar.jsx:207]: done

File: src/js/components/award/shared/InfoTooltipContent.jsx (hunk 2 above)
Comment (chasls2) [reply src/js/components/award/shared/InfoTooltipContent.jsx:1222]: done

File: src/_scss/pages/search/searchSidebar.scss [hunk 8]
```diff
@@ -8,37 +8,79 @@
...
                     width: rem(16);
-                    margin-top: rem(5);
+                    margin-right: rem(8);
+                    & .usa-da-icon-filter {
+                        fill: lighten( $color-gray, 15%);
+                        width: rem(16);
+                        margin-top: rem(4);
+                    }
+                }
+
```
[the remark this answers is not in the document]
Comment (chasls2) [reply src/_scss/pages/search/searchSidebar.scss:49]: done

File: src/_scss/pages/search/searchSidebar.scss (hunk 4 above)
Comment (chasls2) [reply src/_scss/pages/search/searchSidebar.scss:81]: done

File: src/_scss/pages/search/searchSidebar.scss (hunk 5 above)
Comment (chasls2) [reply src/_scss/pages/search/searchSidebar.scss:83]: done

File: src/_scss/pages/search/searchSidebar.scss (hunk 6 above)
Comment (chasls2) [reply src/_scss/pages/search/searchSidebar.scss:62]: done

File: src/_scss/pages/search/searchSidebar.scss [hunk 9]
```diff
@@ -8,40 +8,77 @@
...
+        @include media($small-screen) {
+            display: inline-table;
+            @include align-items(left);
             @include justify-content(flex-start);
             position: relative;
             border-bottom: rem(1) solid $color-gray-lighter;
             height: rem(70);
+            width: 100%;
             margin: 0;
             padding: 0 0 0 rem(15);
```
Comment (brianpetway) [inline src/_scss/pages/search/searchSidebar.scss:23]: Should change this be padding: 0 rem(16);
We need padding right and left now that the text is long enough to hit the right edge, and we should change it from 15 to 16 bc of our new design specs.

20230314FilterTextPaddingRight

File: src/_scss/pages/search/_tooltips.scss [hunk 10]
```diff
@@ -12,6 +12,19 @@ $letter-color: $color-gray-dark;
...
+            padding: rem(16) rem(10);
+            font-weight: bold;
+            font-size: rem(16);
+            width: 100%;
+            color: $letter-color;
+            text-align: left;
+            background: #F0F0F0;
+            line-height: normal;
+            z-index: 2;
+
```
Comment (brianpetway) [inline src/_scss/pages/search/_tooltips.scss:26]: unneeded whitespace

**Comment by anjenkin** [comment]
@chasls2 You need to run `npm install` after making any updates to package.json

File: src/_scss/components/tooltips/_customTooltipContent.scss [hunk 11]
```diff
@@ -76,6 +76,9 @@
         font-style: italic;
     }
 }
+.award-summary__new-category{
```
Comment (brianpetway) [inline src/_scss/components/tooltips/_customTooltipContent.scss:79]: This block can be removed from this file because it's present in searchSidebar.scss and that is applying to the elements on the page

**Comment by brianpetway** [comment]
One more inline comment

File: src/js/components/award/shared/InfoTooltipContent.jsx [hunk 12]
```diff
@@ -1217,6 +1217,47 @@ export const CFDASectionInfo = (
...
+            <p className="award-summary__new-category">
+            The data in award search come primarily from governmentwide award systems in formats called File D1 (for <strong>contract award elements</strong>) and File D2 (for <strong>financial assistance award elements</strong>). If a filter is not listed below, then it applies to both contract and financial assistance award elements.
+            </p>
+            <p>
+            Filters for <strong>agency account elements</strong> apply to both contract and financial assistance award elements, but they come from File C (award data from agency financial systems that are submitted directly to USAspending.gov) rather than from Files D1 or D2.
+            </p>
+            <p className="award-summary__new-category"><strong>Contract Award Elements</strong></p>
+            <p className="award-summary__new-category">
+            Contract award elements contain information from a governmentwide award system called the Federal Procurement Data System (FPDS). This information is extracted by <a href="http://usaspending.gov/">USAspending.gov</a> in a format called File D1.
```
Comment (brianpetway) [inline src/js/components/award/shared/InfoTooltipContent.jsx:1234]: remove hyperlink from USAspending.gov text

**Review by brianpetway — APPROVED** [review]
